### PR TITLE
Added SSE Dummy Proxy.

### PR DESCRIPTION
**Pull request by TheStigger**
Added SSE Dummy Proxy.

### CODE REVIEW

**Comment by Fedorof** [comment]
1. Please, add docstrings.
2. Don't spread related classes among multiple files. It will make code support more difficult. It is better to split big modules later on, when things become more complicated.
3. Don't hardcode the CSV file name.
4. Tetsts!
5. Didn't we agreed to use "EmployeeManager" name?
6. ... \<-- will add in the evening -->

**Comment by ndmytro** [comment]
I completely agree with Alexander's comment regarding the code.

File: sseproxy/proxy.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class ProxyInterface(object):
```
Comment (Fedorof) [inline sseproxy/proxy.py]: It's not an interface, it's an abstract class.

File: sseproxy/sse.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from sseproxy.proxy import ProxyInterface
+
+
+class ProxySSE(ProxyInterface):
```
Comment (Fedorof) [inline sseproxy/sse.py]: This class is useless. Either update or remove it.

File: sseproxy/dummy.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+                          record[ProxyDummy.ROOM_NAME],
+                          record[ProxyDummy.OFFICE_LOCATION]))
+
+    def update_employee(self, employee):
+        ProxyDummy.updater(employee)
+
+    def get_employees(self, employee_id=None, name=None, workplace=None,
+                      room=None, floor=None, project=None, vertical=None,
+                      position=None, location=None):
+        res = self.parser(
```
Comment (Fedorof) [inline sseproxy/dummy.py]: This piece of code is hard to read and hard to understand. Is there a simpler way to filter it?

File: sseproxy/dummy.py [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    SBU_NAME = 1
+    ORGANIZATIONAL_PATH = 2
+    POSITION = 3
+    LAST_NAME = 4
+    PLACE_NUMBER = 5
+    ROOM_NAME = 6
+    OFFICE_LOCATION = 7
+
+    @staticmethod
+    def parse_floor(room):
```
Comment (Fedorof) [inline sseproxy/dummy.py]: This code looks awful.

File: sseproxy/proxy.py [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+    """
+    Interface for Proxy.
+    """
+
+    def get_employees(self, employee_id, name, workplace, room, floor,
+                      project, vertical, position, location):
```
Comment (Fedorof) [inline sseproxy/proxy.py]: Don't make all the arguments required. They should be optional.

File: sseproxy/proxy.py (hunk 1 above)
Comment (Fedorof) [inline sseproxy/proxy.py]: What about "add"?

File: sseproxy/dummy.py [hunk 6]
```diff
@@ -0,0 +1,116 @@
+import csv
+from sseproxy.proxy import ProxyInterface
+
+
+class ProxyDummy(ProxyInterface):
```
Comment (Fedorof) [inline sseproxy/dummy.py]: The base class defines only 2 public methods. Follow it.

File: sseproxy/dummy.py (hunk 3 above)
Comment (TheStigger) [inline sseproxy/dummy.py]: I can do it with kwargs. Will it be OK?

File: sseproxy/dummy.py (hunk 4 above)
Comment (TheStigger) [inline sseproxy/dummy.py]: Do I need to use plain indexes?

**Pull request closed without merging**